src/main/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizeNamesFormatter.java

```diff
@@ -36,6 +36,25 @@ public String getKey() {
 
     @Override
     public String format(String value) {
+        // Handle case names in order lastname, firstname and separated by ","
```

Comment:
For checking for  Curly Braces there is a String Util Method which also has
a Unit Test. Maybe you can reuse that.

2016-06-03 14:13 GMT+02:00 Tobias Diez <notifications@github.com>:

> In
> src/main/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizeNamesFormatter.java
> <https://github.com/JabRef/jabref/pull/1470#discussion_r65697196>:
>
> > @@ -36,6 +36,25 @@ public String getKey() {
> >
> >      @Override
> >      public String format(String value) {
> > +        // Handle case names in order lastname, firstname and separated by ","
>
> test also { as well as ;
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/JabRef/jabref/pull/1470/files/efdced6ec231b82f0f8e82a32a57a2f0fb454fb7#r65697196>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AATi5PDxbzRee0GVpU2oQWEe35bpUgUMks5qIBpNgaJpZM4Is5D7>
> .
>